blog-posts/felix-interview.md

```diff
@@ -0,0 +1,69 @@
+
+
+## Interview mit Felix Hamann
+**This introduction text still to be done**
+
+<br>**Hallo Felix. Schön, dass Du Dir die Zeit nimmst. Stell Dich doch mal kurz vor.**<br>
+Hi! Ich bin Felix. Ich habe noch das Piepsgeräusch der 56k Modems im Ohr, bin also schon eine ganze Weile dabei. Seitdem ich denken kann faszinieren mich Computer und Webentwicklung.
+
+<br>**Du bist der erste Angestellte bei Satellytes. Bist Du nach all den Jahren immer noch happy? Highlights? Lowlights?**<br>
+Immer noch happy. Mit Ausrufezeichen! Highlights: Immer wieder ins kalte Wasser gestoßen zu werden, weil „Life begins at the end of your comfort zone". Beispiele für „kaltes Wasser": Vor kurzem habe ich zum ersten Mal eine Präsentation auf einer Entwickler-Konferenz „eMerge" gehalten und in einem meiner letzten Projekte hatte ich eine Managementrolle inne. Lowlights: Mir fällt eigentlich nur Corona ein, aber ich als introvertiertes Kellerkind bin zu Hause sowieso immer happy... trotzdem finde ich es schade, dass es letztes Weihnachten wieder keine solche legendäre Weihnachtsfeier wie 2019 auf der Hütte geben konnte.
+
+<br>**Du hast den eMerge Talk als Highlight genannt. Wie kam’s dazu?**<br>
+Wir benutzen Lattice als Career Development Tool bei Satellytes und dieser eMerge Talk war eines meiner „Goals". Ich habe mit meinem Manager zusammen beschlossen, dass es eine gute Idee ist das mal auszuprobieren und mich mehr auf die Bühne zu wagen. Das Thema war „Automation mit Github Actions", weil ich in meinem aktuellen Projekt sehr viel damit gemacht habe.
+
+<br>**Und wie war’s?**<br>
+Ich habe zuvor mit meinen Kollegen einige Dryruns gemacht, mit deren Feedbacks ich die Präsentation rund geschliffen habe und am Ende habe ich mich richtig gut damit gefühlt. Beim Talk selbst hatte ich anfangs bestimmt ein Tremolo in meiner Stimme, aber schnell gemerkt, dass ich richtig tief im Thema drinstecke und das daher auch gut erzählen kann. Ich war bei den Proben mit meinen Kollegen aufgeregter als beim Talk selbst. Ich habe sogar die QA-Fragen gut beantworten können vor denen ich ein bisschen Bammel hatte – ich war einfach so gut vorbereitet wie schon lange nicht mehr und das war super!
+
+<br>**Mehr davon?**<br>
+Auf jeden Fall.
+
+<br>**Warum mehr?**<br>
+Neben mir waren Sprecher von Google, Spotify und Microsoft auf der „Bühne", da war ich schon ein bisschen stolz ein Teil davon zu sein. In erster Linie geht es mir aber darum, Wissen weiterzugeben. Kurze kompakte Präsentationen auf Meetups und Konferenzen sind meiner Meinung nach ein idealer Einstieg in diverse Themen. Von allen Entwicklerkonferenzen auf denen ich bisher war bin ich mit einer Liste von Technologien und neuen Methoden nach Hause gekommen, die ich unbedingt ausprobieren wollte. Die Vorstellung, dass meine Präsentation diesen Wunsch in anderen Entwicklern auslöst, fühlt sich klasse an.
+
+![Felix](images/interview/felix-1.jpg)
+
+> Programmieren ist im Wesentlichen, wie ich finde, ein äußerst kreativer Prozess.
```

Comment:
I love the quotes below the image. Helps to bring you much closer to the reader ✨